velox/docs/functions/spark/string.rst

```diff
@@ -390,6 +390,15 @@ String Functions
         SELECT substring_index('aaaaa', 'aa', 5); -- "aaaaa"
         SELECT substring_index('aaaaa', 'aa', -5); -- "aaaaa"
 
+.. spark:function:: to_pretty_string(x) -> varchar
```

Comment:
The document doesn't appear to render properly. Could you take a look?

![Screenshot 2025-03-25 at 15 24 03](https://github.com/user-attachments/assets/97e3821c-ac0f-46a9-a5a8-730667335835)